packages/form_builder_validators/test/form_builder_validators_test.dart

```diff
@@ -162,6 +162,19 @@ void main() {
             expect(validatorAllowEmpty(null), isNull);
             expect(validatorAllowEmpty([]), isNull);
           }));
+  testWidgets(
+      'FormBuilderValidators.lengthEqual for Iterable',
+      (WidgetTester tester) => testValidations(tester, (context) {
+            final validator =
+                FormBuilderValidators.lengthEqual<Iterable<String>>(3);
+            // Pass
+            expect(validator(['one', 'two', 'three']), isNull);
+
+            // Fail
+            expect(validator(null), isNotNull);
+            expect(validator([]), isNotNull);
+            expect(validator(['one', 'two']), isNotNull);
+          }));
```

Comment:
Add more tests with String case

Reply:
The idea is add more tests, not replace that it were.

Tests with String, Iterable and other invalid type

Reply:
I rewrite my test please check.